t/rdf_coordinate_overlap_serializer.t

```diff
@@ -0,0 +1,44 @@
+use strict;
+use Test::More;
+use Test::Deep;
+
+use Bio::EnsEMBL::Mongoose::Serializer::RDFCoordinateOverlap;
+use Bio::EnsEMBL::Mongoose::Persistence::Record;
+use Bio::EnsEMBL::Mongoose::Persistence::RecordXref;
+use FindBin qw/$Bin/;
+
+use IO::String;
+use RDF::Trine;
```

Comment:
You can remove this dependency unless you plan to query the integrity of the data you've written out.